spec/unit/source/git_spec.rb

```diff
@@ -157,9 +187,9 @@
       expect(bn.dirname).to eq 'mybranch'
     end
 
-    it "prepends the source name when prefixing is on" do
-      bn = described_class.new('mybranch', {:prefix => true, :sourcename => 'foo'})
-      expect(bn.dirname).to eq 'foo_mybranch'
+    it "prepends the prefix name when prefixing is overridden" do
```

Comment:
The tests that articulate this behaviour are found here: 

https://github.com/brettswift/r10k/blob/feature/master/multi_tenant_prefix/spec/unit/source/base_spec.rb#L33-L50

You'll see that the only output is false or string. 

It's a little bit unintuitive than my original fix which was done at the `source/git.pp`  but doing it at `source/base.pp` is a little more appropriate I think.

What do you think about this flow?